tests/background_workload_test.py

```diff
@@ -0,0 +1,176 @@
+# Copyright 2015 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for background workload"""
+
+import unittest
+import os
+import mock
+from mock import patch
+from tests import mock_flags
+from perfkitbenchmarker import configs
+from perfkitbenchmarker.linux_benchmarks import ping_benchmark
+from perfkitbenchmarker import benchmark_spec
+from perfkitbenchmarker import timing_util
+from perfkitbenchmarker import pkb
+
+
+NAME = 'ping'
+UID = 'name0'
+
+CONFIG_WITH_BACKGROUND_CPU = """
+ping:
+    description: Benchmarks ping latency over internal IP addresses
+    vm_groups:
+      vm_1:
+        vm_spec:
+          GCP:
+            machine_type: n1-standard-1
+      vm_2:
+        vm_spec:
+          GCP:
+            background_cpu_threads: 3
+            machine_type: n1-standard-1
+"""
+
+
+class TestBackgroundWorkload(unittest.TestCase):
+
+  def testBackgroundWorkloadInPKB(self):
+    """ Check that benchmark spec methods are called by pkb"""
+    with mock_flags.PatchFlags() as mocked_flags:
+
+      mocked_flags.os_type = 'debian'
+      mocked_flags.machine_type = None
```

Comment:
This None assignment should be unnecessary now because of the new mocked default value.